jobs/build/coreos-installer_sync/Jenkinsfile

```diff
@@ -42,6 +43,23 @@ node {
                     defaultValue: "",
                     trim: true,
                 ),
+                choice(
+                    name: 'ENV',
+                    description: 'Which environment to sign in',
+                    choices: [
+                        "stage",
+                        "prod",
+                    ].join("\n"),
+                ),
+                choice(
+                    name: 'KEY_NAME',
+                    description: 'Which key to sign with\nIf ENV==stage everything becomes "test"\nFor prod we currently use "redhatrelease2"',
+                    choices: [
+                        "test",
+                        "beta2",
+                        "redhatrelease2",
+                    ].join("\n"),
+                ),
```

Comment:
I don't think those low-level signing parameters should be exposed to the user. Based on the requirement, this should always be signed with `redhatrelease2` key.

Reply:
updated with limited choise